WordPress/src/main/java/org/wordpress/android/ui/DeepLinkingIntentReceiverActivity.java

```diff
@@ -89,8 +93,18 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     private void showPost() {
         if (!TextUtils.isEmpty(mBlogId) && !TextUtils.isEmpty(mPostId)) {
             try {
-                ReaderActivityLauncher.showReaderPostDetail(this, mIsFeed, Long.parseLong(mBlogId),
-                        Long.parseLong(mPostId), false);
+                final long blogId = Long.parseLong(mBlogId);
```

Comment:
Good point! Thanks for the pointer!

I was just reusing the code already in place in this case but yeah, it could probably be simplified by using `StringUtils.stringToLong()`.